README.md

```diff
@@ -231,17 +231,13 @@ To run the integration tests, which test the integration between this node modul
 [ilp-plugin-bells](https://github.com/interledgerjs/ilp-plugin-bells), run:
 
 ```sh
-$ npm run integration-loader
+$ node ./node_modules/.bin/integration-loader
 $ node ./node-modules/.bin/integration setup
 $ node ./node_modules/.bin/integration test
 ```
 
-To repeat one of the integration tests, run for instance:
-
-```sh
-$ cd integration-test
-$ mocha ../node_modules/five-bells-integration-test/src/tests/index.js
-```
+After a fix, you can repeat just the last step, or just the two last steps. It will complete faster than
```

Comment:
I think you need to run `rm -rf integration-test/data` if you want to re-run `integration test`